src/components/pages/Navigation.tsx

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
+import { BrowserRouter, Route, Switch, Link } from 'react-router-dom';
+import NameForm from './StatedNameForm';
+import GraphForm from './StatedGraphForm';
 
 const Navigation = () => {
   return (
     <div>
-      <NavLink to="/">Home </NavLink>
-      <NavLink to="/graph">graph</NavLink>
+      <h2>
+        <BrowserRouter>
+          <h2>Welcome to cashburn, app for budget management</h2>
+          <li>
+            <Link to="/" className="nav-link">
+              Home
+            </Link>
+          </li>
+          <li>
+            <Link to="/graph" className="nav-link">
```

Comment:
Użycie znacznika li pozwala na lepsze wyeksponowanie linków, które przy użyciu Navlink prawie zlewały się ze sobą. Fajnie by było to zaimplementować w postaci panelu z menu nawigacji na górze. Również sprawa na przedyskutowanie.